packages/commonwealth/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+services:
+  cw_postgres:
+    container_name: cw_postgres
+    image: postgres:15.1-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: commonwealth
+      POSTGRES_PASSWORD: edgeware
+      POSTGRES_DB: commonwealth
+    volumes:
+      - .postgres:/var/lib/postgresql/data
```

Comment:
We need to add these volumes to the gitignore, otherwise they will be able added to git staging. So we should be adding the following lines to the packages/commonwealth/.gitignore:
```
# Docker volumes
.rabbitmq
.postgres
.redis-data
```